config.md

```diff
@@ -95,6 +95,7 @@ See links for details about [mountvol](http://ss64.com/nt/mountvol.html) and [Se
 **`process`** (object, required) configures the container process.
 
 * **`terminal`** (bool, optional) specifies whether you want a terminal attached to that process, defaults to false.
+* **`initialConsoleSize`** (box, optional) specifies the initial console size of the terminal if attached.
```

Comment:
I prefer naming as `consoleWindowSize` as @laijs suggested, because we might want to update process console size and it seems weird to assign new config to the field with `initial` prefixed.